content/attacks/posts/23-07-30-CurveFi

```diff
@@ -0,0 +1,68 @@
+
+As illustrated below, the exploiter has been returning some of the funds. 
+
+Alechemix, another DeFi lending platform, saw its alETH/ETH lose 7,258.70 WETH and 4,821.55 alETH, worth a total of $22.6 million. pETH/ETH, associated with NFT lending project JPEG’d, was drained of 6,106.65 in WETH, roughly $11 million at the time. 
+
+After an initial failed [attack](https://etherscan.io/tx/0xe2eb61b1ff11e447b791e030039ce89ae294423da1622d0be8c7ec3b2ff1a22a) from seemingly the same hacker, white hat hacker c0ffeebabe deployed copycat code and [frontran](https://etherscan.io/tx/0xc93eb238ff42632525e990119d3edc7775299a70b56e54d83ec4f53736400964) that hack on msETH/ETH, draining 959.7 msETH and 866.55 ETH, worth just under $1.5 million which Metronome Synth has said it considers recovered. Another copycat blackhat seems to have swooped in to steal the remaining 435.86 msETH in that pool. Metronome Synth runs an app that mints synthetic tokens.
+
+Curve’s own CRV/ETH pool lost 7,193,401.77 CRV, 7,680.49 WETH, and 2,879.65 ETH, for a total of over $24 million as of the time of the hack. The project has taken the biggest hit. Total Value Locked in the protocol slipped from $3.3 billion to $2.3 billion, according to [DeFi Llama](https://defillama.com/protocol/curve-finance). Its governance token, CRV, [lost](https://coinmarketcap.com/currencies/curve-dao-token/) about 40% of its value in the weeks following the attack.
+
+## Timeline: 
+
+July 30: 
+
+UTC 13:10 First exploit, on pETH/ETH pool, [begins](https://etherscan.io/tx/0xa84aa065ce61dbb1eb50ab6ae67fc31a9da50dd2c74eefd561661bfce2f1620c).
+
+UTC 14:50: [Attack](https://etherscan.io/tx/0xc93eb238ff42632525e990119d3edc7775299a70b56e54d83ec4f53736400964) on msETH/ETH pool.  
+
+UTC 15:34: [Attack](https://etherscan.io/tx/0xb676d789bb8b66a08105c844a49c2bcffb400e5c1cfabd4bc30cca4bff3c9801) on alETH/ETH pool. 
+
+UTC 16:44: Vyper reports vulnerability, followed [immediately](https://twitter.com/CurveFinance/status/1685693202722848768) by Curve identifying targeted pools other than CRV/ETH. 
+
+UTC 19:16 and 19:30: Hacker pulls WETH and then CRV from pool
+
+July 31: UTC 09:49: c0ffeebabe.eth makes return to Metronome Synth. 
+
+Aug. 4: JPEG’d confirms return of funds from the hacker. 
+
+Aug. 11: Curve [puts out](https://twitter.com/CurveFinance/status/1689995987073261568) post-hack safety report, but also [says](https://twitter.com/CurveFinance/status/1689995852411027456) 70% of funds affected have been recovered. 
+
+Aug. 18: Metronome Synth [releases](https://metronomedao.medium.com/vyper-curve-exploit-post-mortem-6a42d6d9fa07) its own post-mortem. 
+
+## Attackers: 
+
+Unknown origins; Curve Finance has offered a $1.85 million bounty to anyone who can identify the hacker. 
+
+Just under $16 million of the hacked funds are still sitting in 0xb1c33b391c2569b737ec387e731e88589e8ec148 as of 25 08 2023. 
```

Comment:
date format should read 25 AUG 2023

Reply:
Ah, misunderstood that. Have changed; all caps for the month right?